spec/models/crop_spec.rb

```diff
@@ -328,6 +328,8 @@
   context "search" do
     before :each do
       @mushroom = FactoryGirl.create(:crop, :name => 'mushroom')
+      @mushroom.__elasticsearch__.index_document
+      sleep 1
```

Comment:
Yeah, I didn't like the <code>sleep</code> either. Thanks both of you for those comments, they reinforced my hate so much that I fixed it! Replacing the sleep with <code>Crop.__elasticsearch__.refresh_index!</code> has worked.

I'm also adding some code under spec/support so it will recreate/clean index before each test.